tests/benchmarks/power_consumption/gpio/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  tags: ppk_power_measure ci_tests_benchmarks_current_consumption
+
```

Comment:
Please add `integration_platforms` section, which is mandatory for testcase / sample yamls.